vaadin-testbench-unit-shared/src/main/java/com/vaadin/flow/component/tabs/TabsTester.java

```diff
@@ -54,8 +55,12 @@ public void select(String label) {
      */
     public void select(int index) {
```

Comment:
I would perhaps update the JavaDoc to underline the case that non visible tabs are not calculated into the index when selecting.